website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -345,6 +345,8 @@ models:
 
 </File>
 
+Please note that in order to apply this, [persist_docs](https://docs.getdbt.com/reference/resource-configs/persist_docs) should be enabled on your dbt_project.yml file.
```

Comment:
Two notes:
- I think it needs to be `persist_docs: {columns: True}`
- Could be enabled from `dbt_project.yml` or within the model/snapshot `config()`, we can just link to the other docs for that piece

```suggestion
Please note that in order for policy tags to take effect, [column-level `persist_docs`](https://docs.getdbt.com/reference/resource-configs/persist_docs) must be enabled for the model, seed, or snapshot.
```